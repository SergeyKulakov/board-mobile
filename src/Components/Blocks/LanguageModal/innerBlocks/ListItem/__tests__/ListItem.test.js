import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import Component from '../ListItem'

describe('LanguagesModal.ListItem', () => {
  const defaultProps = {
    t: text => text,
    item: {
      type: 'en',
      flag: 'english',
      name: 'English',
    },
    onPressItem: () => null,
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
