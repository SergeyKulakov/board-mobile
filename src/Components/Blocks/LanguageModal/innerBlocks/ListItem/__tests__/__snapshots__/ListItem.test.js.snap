// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LanguagesModal.ListItem render snapshot 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  onPress={[Function]}
>
  <Styled(View)>
    <Styled(View)
      isLeft={true}
    />
    <Styled(View)>
      <Component
        fileName="english"
      />
      <Styled(Text)>
        languages.English
      </Styled(Text)>
    </Styled(View)>
    <Styled(View) />
  </Styled(View)>
</TouchableOpacity>
`;
